src/InfluxDB/Point.php

```diff
@@ -81,26 +88,39 @@ public function __toString()
         return $string;
     }
 
+    /**
+     * @param  array $arr
+     * @return string
+     */
     private function arrayToString(array $arr)
     {
         $strParts = array();
 
         foreach ($arr as $key => $value) {
-            $strParts[] = "{$key}={$value}";
+            $strParts[] = sprintf('%s=%s', $key, $value);
```

Comment:
So you agree that more stuff should be changed:

- https://github.com/influxdb/influxdb-php/blob/fd80291e151c4eba966f85165082fc920c111670/src/InfluxDB/Client.php#L264
- https://github.com/influxdb/influxdb-php/blob/fd80291e151c4eba966f85165082fc920c111670/src/InfluxDB/Client.php#L143 (wth?)
- https://github.com/influxdb/influxdb-php/blob/fd80291e151c4eba966f85165082fc920c111670/src/InfluxDB/Client.php#L178
- https://github.com/influxdb/influxdb-php/blob/fd80291e151c4eba966f85165082fc920c111670/src/InfluxDB/Client.php#L208

And many others.